ocs_ci/ocs/ui/page_objects/buckets_tab.py

```diff
@@ -168,7 +139,7 @@ def create_folder_in_bucket(
         self.do_send_keys(self.bucket_tab["folder_name_input"], folder_name)
         self.do_click(self.bucket_tab["submit_button_folder"])
 
-        folder_path = self.generate_folder_with_file()
+        folder_path = generate_folder_with_files(num_files=400)
```

Comment:
any reason for 400? I see the default in `generate_folder_with_files` is 300

Reply:
object paging is 300, so it need to be highrt. i can remove that default